src/index.js

```diff
@@ -0,0 +1,55 @@
+import './style.css';
+
+const list = [
+  {
+    description: 'Finish javascript project',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Go to the supermarket',
+    completed: false,
+    index: 1,
+  },
+];
+
+// Query selectors
+const listContainer = document.querySelector('.content');
+
+// Functions
+function createList(task) {
+  const listItem = document.createElement('div');
+  listItem.className = 'list';
+
+  const eachTasks = document.createElement('div');
+  eachTasks.className = 'eachtask';
+  listItem.appendChild(eachTasks);
+
+  const box = document.createElement('input');
+  box.type = 'checkbox';
+  box.className = 'checkbox';
+  eachTasks.appendChild(box);
+
+  const newTask = document.createElement('div');
+  newTask.className = 'text';
+  newTask.textContent = task.description;
+  eachTasks.appendChild(newTask);
+
+  const btn = document.createElement('button');
+  btn.type = 'button';
+  btn.className = 'btn-right';
+  listItem.appendChild(btn);
+
+  listContainer.appendChild(listItem);
+  return listItem;
+}
+
+list.reverse().forEach((task) => createList(task));
```

Comment:
- [ ] The list items are supposed to be in order, instead of reverse order, so from 0, 1, 2... which is not achieved using reverse 

![image](https://user-images.githubusercontent.com/73140461/149182947-74919de4-42b9-44ac-bab0-0c59d330ff13.png)

Reply:
removed and working